src/core/LiterallyCanvas.coffee

```diff
@@ -313,7 +320,7 @@ module.exports = class LiterallyCanvas
     return unless shapes.length
     drawShapes = =>
       for shape in shapes
-        renderShapeToContext(ctx, shape, {retryCallback})
+        shape.draw(ctx, { retryCallback })
```

Comment:
Only because it seemed to me that since we were giving shapes the ability to draw themselves we might as well use it rather than a parallel call to renderShapeInContext. I did that while I was playing with rendering and possibly wanting to change it a bit... 

With this change, the shapes use the canvas renderer but LC doesn't have to know about it any more. It seemed cleaner to me but I'm not overly attached to it if there is a good reason to keep it around in LC.